instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientAttributesGetter.java

```diff
@@ -20,9 +20,23 @@
  */
 public interface DbClientAttributesGetter<REQUEST> extends DbClientCommonAttributesGetter<REQUEST> {
 
+  /**
+   * @deprecated Use {@link #getDbQueryText(REQUEST)} instead.
+   */
+  @Deprecated
   @Nullable
   String getStatement(REQUEST request);
```

Comment:
since this was just a rename, I think could add default implementation that delegates to `getDbQueryText`, then wouldn't need to override this everywhere

Reply:
it's better to use @deprecate so that when old semconv is removed, we can use it to clean it up.

Reply:
you can still keep `@Deprecated` on the default implementation

Reply:
wdyt about implementing it the same way as Mateusz did with http semconv in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8632. Existing methods that are renamed are changed to deprecated default methods, new methods are introduced as default methods that delegate to the old deprecated method with a `// TODO: make this required to implement` comment. This would allow doing the renames without completely breaking existing implementations.

Reply:
This would also let you implement extractors so that they call only the new methods and in the getter implementations you could also implement only the new methods.